### PR TITLE
added several new samples and articles

### PR DESCRIPTION
**Pull request by romankhar**
also made some changes to the sub-titles of sections

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,11 +39,24 @@
 - [Updates to OpenWhisk Node.js actions](https://medium.com/openwhisk/updates-to-openwhisk-node-js-actions-ed5556cd5ae9) - Recent changes including deprecation for the GA release.
 
 
+## Tutorials
+
+*Tutorials and other material to help you learn OpenWhisk*
+
+- [Official OpenWhisk Tutorial](https://github.com/openwhisk/openwhisk-tutorial) - Interactive tutorials for learning how to use OpenWhisk.
```
Comment (rabbah) [inline README.md]: Tutorial and later tutorials. Should both be plural?

File: README.md [hunk 2]
```diff
@@ -39,11 +39,24 @@
...
+
 
-## Applications
+## Sample Applications
 
 *Sample open-source projects built using the OpenWhisk platform*
 
+- [OpenWhisk 101 - Message Hub and Kafka Data Processing](https://github.com/IBM/openwhisk-data-processing-message-hub) - Example of using OpenWhisk with Message Hub and Kafka to consume and publish messages.
+- [Transit IoT](https://medium.com/openwhisk/transit-flexible-pipeline-for-iot-data-with-bluemix-and-openwhisk-4824cf20f1e0) - Comprehensive example of using OpenWhisk for IoT data processing that uses Docker, Node-RED, Message Hub (Kafka based), Object Storage, Spark and Bluemix Data Science Experience services for data analytics.
+- [Logistics Wizard](https://www.ibm.com/blogs/bluemix/2017/02/microservices-multi-compute-approach-using-cloud-foundry-openwhisk/) is an enterprise-grade sample application which leverages OpenWhisk and CloudFoundry to build 12-factor style application. It is a smart supply chain management solution that aims to simulate an environment running an ERP system.
```
Comment (rabbah) [inline README.md]: application -> applications?

File: README.md [hunk 3]
```diff
@@ -39,11 +39,24 @@
...
+- [Web Actions](https://medium.com/openwhisk/web-actions-serverless-web-apps-with-openwhisk-f21db459f9ba) - This sample shows hot to use OpenWhisk to build a complete Web App.
```
Comment (rabbah) [inline README.md]: hot -> how.

File: README.md [hunk 4]
```diff
@@ -39,11 +39,24 @@
...
+- [Dark vision](https://github.com/IBM-Bluemix/openwhisk-darkvisionapp) - application that shows how to use OpenWhisk, Cloudant, Watson Visual Recognition, Object Storage to process video frames, tag and recognize scenes.
```
Comment (rabbah) [inline README.md]: capitalize "a" in "application".

Comment (rabbah) [reply README.md]: same below for "d" in "demonstrates"

File: README.md (hunk 1 above)
Comment (romankhar) [reply README.md]: fixed it (it was already there :-) )

File: README.md (hunk 2 above)
Comment (romankhar) [reply README.md]: changed to "12-factor style applications"

File: README.md (hunk 3 above)
Comment (romankhar) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (romankhar) [reply README.md]: done

**Comment by romankhar** [comment]
all fixed and commited

**Review by jthomas — CHANGES_REQUESTED** [review]
I want to keep the "applications" section with links to Github repositories. 
Can you move the links that point to Medium blog posts to the articles section?

**Comment by jthomas** [comment]
@mrutkows can you give me write access to this repo? I've filed my CLA.

**Comment by romankhar** [comment]
every one of those Medium articles has associated Repo with a complete sample - do we need to list it twice - once under Articles and once under Samples? I would rather have it under Samples since most people wont look at the list of articles when there are looking for a sample

**Review by csantanapr — CHANGES_REQUESTED** [review]
I would like to see Sample Applications sections pointing to github repos. in addition a link to the companion blog/article should be added.
Each application bullet ideally would have two links the primary one to the repo, and secondary (optional) to the companion blog/article.

Also we should check if the repo itself should also have a link to the companion article.

In the Article section, should only be links to blog/article that don't have associated repo. and only a link to the article.

**Comment by romankhar** [comment]
I replaced links under samples with links to GitHub and moved those without GitHub to "articles". Please take a look at this and feel free to edit directly - as soon as we are all done editing it, I will copy&paste into my fork and refresh the PR: https://docs.google.com/document/d/1PIYg1sUDV8wimnm5H9imVJ7ZPXEGLzD3LhW4qKzkQ_Y/edit?usp=sharing

**Comment by jthomas** [comment]
LGTM.

**Review by csantanapr — APPROVED** [review]

**Pull request merged**